docs/source/reference/server-api.md

```diff
@@ -119,67 +101,50 @@ while the server is not ready yet:
   }
 ```
 
-Note that `ready` is false and `pending` is `spawn`.
-This means that the server is not ready
-(attempting to access it may not work)
-because it isn't finished spawning yet.
-We'll get more into that below in [waiting for a server][].
+Note that `ready` is assigned `false` and `pending` is assigned `spawn`, meaning that the server is not ready and attempting to access it may not work as it is still in the process of spawning.
 
 [waiting for a server]: waiting
 
 (starting)=
 
 ## Starting servers
 
-To start a server, make the request
+To start a server, make the following API request:
 
 ```
 POST /hub/api/users/:username/servers/[:servername]
 ```
 
 **Required scope: `servers`**
 
-(omit servername for the default server)
-
-Assuming the request was valid,
-there are two possible responses:
+Assuming the request was valid, there are two possible responses:
 
 201 Created
-: This status code means the launch completed and the server is ready.
-It should be available at the server's URL immediately.
+: This status code means the launch completed and the server is ready and is available at the server's URL immediately.
 
 202 Accepted
-: This is the more likely response,
-and means that the server has begun launching,
-but isn't immediately ready.
-The server has `pending: 'spawn'` at this point.
-
-_Aside: how quickly JupyterHub responds with `202 Accepted` is governed by the `slow_spawn_timeout` tornado setting._
+: This is the more likely response, and means that the server has begun launching,
+but is not immediately ready. As a result, the server shows `pending: 'spawn'` at this point and you should wait for it to start.
 
 (waiting)=
 
-## Waiting for a server
+## Waiting for a server to start
 
-If you are starting a server via the API,
-there's a good change you want to know when it's ready.
-There are two ways to do with:
+After receiving a `202 Accepted` response, you have to wait for it to start. Two approaches can be applied to establish when the server is ready:
 
 1. {ref}`Polling the server model <polling>`
-2. the {ref}`progress API <progress>`
+2. {ref}`Using the Progress API <progress>`
 
 (polling)=
 
 ### Polling the server model
 
-The simplest way to check if a server is ready
-is to request the user model.
+The simplest way to check if a server is ready is to programmatically query the server model until the following two conditions are true:
 
-If:
+: 1. The server name is contained in the `servers` response, and
```

Comment:
These colons are meant for definition lists. I suspect they should be removed. You can check the resulting rendered docs [here](https://jupyterhub--4106.org.readthedocs.build/en/4106/reference/server-api.html#polling-the-server-model).

Reply:
Thank you.